addons-l10n/en/variable-manager.json

```diff
@@ -1,7 +1,12 @@
 {
-  "variable-manager/for-this-sprite": "Variables for this sprite",
-  "variable-manager/for-all-sprites": "Variables for all sprites",
+  "variable-manager/vars-for-this": "Variables for this sprite",
+  "variable-manager/lists-for-this": "Lists for this sprite",
```

Comment:
You should avoid renaming message since they'll appear as entirely new strings to translators.